main.py

```diff
@@ -1,11 +1,23 @@
 import flask
 import random
+import os
+from flask import Flask, flash, redirect, render_template, request, session, abort
 
 app = flask.Flask(__name__)
 
 @app.route("/")
-def index():
-    return 'Collaborative DJing WebApp!'
+def main():
+    #if not session.get('logged_in'):
+        #return render_template('login.html')
+    #else:
+    return flask.render_template('login.html')
 
-if __name__ == '__main__':
-    app.run(debug=True)
\ No newline at end of file
+@app.route('/login', methods=['POST'])
+def login():
+    if flask.request.form['psw'] == 'password' and flask.request.form['email'] == 'admin':
```

Comment:
Was this tested?  When I pull your code and try to open the login page, I get errors.

Reply:
I’m taking a look at it now.  I think It works when it’s in the first route. But if you try to add login after the numbers i don’t think it will work. I think it’s something to do with connecting the pages. We plan on making it so that we’re able to go to the login page from the soon to be header from the landing page. We will also make that statement mentioned return the playlist page.